ddev/src/ddev/e2e/agent/docker.py

```diff
@@ -86,17 +86,17 @@ def start(self, *, agent_build: str, local_packages: dict[Path, str], env_vars:
         if not agent_build:
             agent_build = 'datadog/agent-dev:master'
 
-        # Add a potentially missing `py` suffix for default non-RC builds
-        if (
-            'rc' not in agent_build
-            and 'py' not in agent_build
-            and agent_build != 'datadog/agent:6'
-            and agent_build != 'datadog/agent:7'
-        ):
-            agent_build = f'{agent_build}-py{self.python_version[0]}'
-
-        if self.metadata.get('use_jmx') and not agent_build.endswith('-jmx'):
-            agent_build += '-jmx'
+            # Add a potentially missing `py` suffix for default non-RC builds
+            if (
+                'rc' not in agent_build
+                and 'py' not in agent_build
+                and agent_build != 'datadog/agent:6'
+                and agent_build != 'datadog/agent:7'
+            ):
+                agent_build = f'{agent_build}-py{self.python_version[0]}'
+
+            if self.metadata.get('use_jmx') and not agent_build.endswith('-jmx'):
+                agent_build += '-jmx'
```

Comment:
Well, in fact it does not really makes sense to move this inside the `if` statement sice the `agent_build` is hardcoded 🤔